configs/retinaface/retinaface_r152_fpn_ssh_widerface.py

```diff
@@ -0,0 +1,184 @@
+_base_ = [
```

Comment:
Please refer to https://github.com/open-mmlab/mmdetection/blob/master/configs/wider_face/ssd300_wider_face.py format to modify the configuration file